workspaces/quickstart/plugins/quickstart/src/components/QuickstartContent/QuickstartCtaLink.tsx

```diff
@@ -16,13 +16,15 @@
 import OpenInNewIcon from '@mui/icons-material/OpenInNew';
 import { QuickstartItemCtaData } from '../../types';
 import { LinkButton } from '@backstage/core-components';
+import { useTranslation } from '../../hooks/useTranslation';
 
 export type QuickstartCtaLinkProps = {
   cta?: QuickstartItemCtaData;
   onClick: () => void;
 };
 
 export const QuickstartCtaLink = ({ cta, onClick }: QuickstartCtaLinkProps) => {
+  const { t } = useTranslation();
```

Comment:
A little bit below is a left over english string "Got it!"

Can you translate this fallback in this PR as well?